FEATURE_MATRIX.md

```diff
@@ -0,0 +1,65 @@
+# Kernel Feature Matrix by Language
+
+## AI Services
+| | C# | Python | Notes |
+|---|---|---|---|                   
+| TextGeneration                    | ✅ | ✅ | Example: Text-Davinci-003 |
+| TextEmbeddings                    | ✅ | ✅ | Example: Text-Embeddings-Ada-002 |
+| ChatCompletion                    | ✅ | ❌ | Example: GPT4, Chapt-GPT |
+| Image Generation                  | ✅ | ❌ | Example: Dall-E |
+
+## AI Service Endpoints
+| | C# | Python | Notes |
+|---|---|---|---|                   
+| OpenAI                            | ✅ | ✅ | |
+| AzureOpenAI                       | ✅ | ✅ | |
+| Hugging Face                      | ✅ | ❌ | Coming soon to Python - both native and web endpoint support | 
+| Custom                            | ✅ | ❌ | Requires the user to define the service schema in their application |
+
+## Tokenizers
+| | C# | Python | Notes |
+|---|---|---|---|         
+| GPT2                              | ✅ | 🔄 | Can be manually added to Python via the `transformers` library |
+| GPT3                              | ✅ | ❌ | | 
+| tiktoken                          | 🔄 | ❌ | Coming soon to Python |         
```

Comment:
coming soon to Python and C#

Reply:
python: can be added manually with `pip install tiktoken`

Reply:
added